components/checkbox/index.en-US.md

```diff
@@ -0,0 +1,31 @@
+---
+category: Components
+type: Form Controls
+title: Checkbox
+---
+
+Checkbox.
+
+## When To Use
+
+- Used to select serveral values in multiple options.
+- If you just use only one checkbox, it is the same as `Switch` to switch between two states. The difference is that `Switch` will trigger state change directly, but `Checkbox` just marks state changed and need to be submitted.
```

Comment:
- If you use only one checkbox, it is the same as using `Switch` to toggle between two states. The difference is that `Switch` will trigger state change directly, but `Checkbox` just marks the state as changed and this needs to be submitted.